packages/calypso-e2e/src/data-helper.ts

```diff
@@ -113,15 +113,44 @@ export function randomPhrase(): string {
  * Generates a structured test suite name from environmental and user-provided parameters.
  *
  * @param {string} title Title of the test.
+ * @param {{key: string}: string} param0 Object parameter.
+ * @param {boolean} param0.parallel Whether to include the @parallel tag. Defaults to true.
  * @returns {string} Test suite name.
  */
-export function createSuiteTitle( title: string ): string {
+export function createSuiteTitle(
+	title: string,
+	{ parallel = true }: { parallel?: boolean } = {}
+): string {
 	const parts = [
 		`[${ getJetpackHost() }]`,
 		`${ toTitleCase( title ) }:`,
 		`(${ getViewportName() })`,
-		'@parallel',
 	];
 
+	if ( parallel ) {
+		parts.push( '@parallel' );
+	}
+
+	return parts.join( ' ' );
+}
+
+/**
+ * Generates a structured test subsuite name.
+ *
+ * @param {string} title Title of the subsuite.
+ * @param {{key: string}: string} param0 Object parameter.
+ * @param {boolean} param0.parallel Whether to include the @parallel tag. Defaults to false.
+ * @returns {string} Test subsuite name.
+ */
+export function createSubsuiteTitle(
```

Comment:
Changes in this file are meant to provide better control to test writers on exactly where the `@parallel` tag should be placed.

Previously the `@parallel` tag was always placed at the parent suite level.

Reply:
I had thought we still support subsuite with `@parallel`; if that's not the case, then I wonder if I should split these tests up into individual files then so they run a lot faster.

Reply:
Removed all subsuite with `@parallel`.